app/Providers/AuthServiceProvider.php

```diff
@@ -13,16 +13,14 @@ class AuthServiceProvider extends ServiceProvider
      * @var array<class-string, class-string>
      */
     protected $policies = [
-        'App\Models\Model' => 'App\Policies\ModelPolicy',
+        // 'App\Models\Model' => 'App\Policies\ModelPolicy',
```

Comment:
Why comment it out?

Reply:
It was triggering an error if not, to be honest I am not using it in my own project so I never checked how policies are working